docker/letsencrypt/letsencrypt-certbot/haproxy-reload.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh 
+
+# Update HAProxy with new certs via its API
+
+le_base=/etc/letsencrypt/live
+if [[ -d "$le_base" ]]; then
```

Comment:
It's my understanding that `[[` is a bash-ism (and other shells, but notably is not part of POSIX). Perhaps just change the shell invocation at the top to `#!/bin/bash`.

Reply:
Done.